docs/source/030_installing.rst

```diff
@@ -96,17 +97,19 @@ On Ubuntu Linux, the following packages will work::
 Note; Centos may require looking the EPEL repo.  At TACC we install the
 following rpms::
 
-   lua-posix-32-2.el7.x86_64
-   lua-term-0.03-3.el7.x86_64
+   lua-5.1.4-15.el7.x86_64
    lua-bitop-1.0.2-3.el7.x86_64
-   lua-filesystem-1.6.2-2.el7.x86_64
    lua-devel-5.1.4-15.el7.x86_64
-   lua-lpeg-0.12-1.el7.x86_64
-   lua-5.1.4-15.el7.x86_64
+   lua-filesystem-1.6.2-2.el7.x86_64
    lua-json-1.3.2-2.el7.noarch
-   tcl-devel-8.5.13-8.el7.x86_64 
+   lua-lpeg-0.12-1.el7.x86_64
+   lua-posix-32-2.el7.x86_64
+   lua-term-0.03-3.el7.x86_64
+
+You will also need the tcl and tcl-devel packages as well.::
 
-You will also need the libtcl and tcl packages as well.
+   tcl-8.5.13-8.el7.x86_64
+   tcl-devel-8.5.13-8.el7.x86_64 
```

Comment:
tcl-devel is only needed if you want to use the binary tcl parser. Lmod can work without it.

Reply:
Thanks for the clarification.  Should I amend the text to say so?  It was listed in Robert's original list of what is installed at TACC, so I kept it in the list.

Reply:
I think that tcl-devel is only needed when you compile Lmod. For running you only need tcl itself.

@rtmclay correct?

Reply:
I just check the RPM specs we use (and the one from Fedora):

```
BuildRequires:  gcc
BuildRequires:  lua-devel
BuildRequires:  lua-filesystem
BuildRequires:  lua-json
BuildRequires:  lua-posix
BuildRequires:  lua-term
BuildRequires:  tcl-devel
BuildRequires:  zsh
Requires:       lua-filesystem
Requires:       lua-json
Requires:       lua-posix
Requires:       lua-term
Requires:       /usr/bin/ps
```

And a quick test confirmed: if you have `--with-fastTCLInterp=no`, you don't need tcl-devel or lua-devel. Else you need both but just for compiling Lmod.

Reply:
Ward, I suspect that because this precedes a section on running `./configure` to install, that they should probably both remain in there.  Up to Robert, if the change is accepted at all.

Reply:
Sure, I would just add that you only need these to build Lmod. For just using/running Lmod, no -devel packages are needed.